### PR TITLE
ansible/workshops: Have the functional test run by zuul

### PR DESCRIPTION
**Pull request by Spredzy**
Depends-On: ansible/project-config#385

### CODE REVIEW

**Comment by Spredzy** [comment]
recheck

File: zuul.d/jobs.yaml [hunk 1]
```diff
@@ -745,3 +745,36 @@
     <<: *ansible-tox-molecule-base
     name: molecule-tox-py38
     parent: ansible-tox-py38
+
+
+# Ansible Workshops jobs
+#
+- job:
+    name: workshop-f5
```
Comment (pabelanger) [inline zuul.d/jobs.yaml]: workshops-f5-integration ?

File: zuul.d/jobs.yaml [hunk 2]
```diff
@@ -745,3 +745,36 @@
...
+    parent: ansible-workshop
+    vars:
+      workshop_type: f5
+
+- job:
+    name: workshop-networking
```
Comment (pabelanger) [inline zuul.d/jobs.yaml]: same format for all jobs

**Comment by pabelanger** [comment]
recheck

**Pull request merged**